Dockerfile

```diff
@@ -112,660 +114,631 @@ RUN set -x; \
 	&& cd $MW_HOME/extensions/Variables \
-	&& git checkout -q e20f4c7469bdc724ccc71767ed86deec3d1c3325 \
-	# VEForAll (v. 0.4)
+	&& git checkout -q b4a9063f16a928567e3b6788cda9246c2e94797f \
+	# VEForAll (v. 0.4.1)
 	&& git clone --single-branch -b master https://gerrit.wikimedia.org/r/mediawiki/extensions/VEForAll $MW_HOME/extensions/VEForAll \
 	&& cd $MW_HOME/extensions/VEForAll \
-	&& git checkout -q d0aec153e80b6604739aeffb60381f52d921db51 \
+	&& git checkout -q 2f1f08eca7fbf61198e5f4ccf2d627a6c9ef7b64 \
 	# VoteNY
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/VoteNY $MW_HOME/extensions/VoteNY \
 	&& cd $MW_HOME/extensions/VoteNY \
-	&& git checkout -q b73dd009cf151a9f442361f6eb1e355817ca1e18 \
+	&& git checkout -q 11c103f4b9167a8d8d5e850d8a781c6f49b249c1 \
 	# WhoIsWatching
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/WhoIsWatching $MW_HOME/extensions/WhoIsWatching \
 	&& cd $MW_HOME/extensions/WhoIsWatching \
-	&& git checkout -q 510e95a76fe140890ea83abf75be64ce97f7fd30 \
+	&& git checkout -q 836a31018e26ab7c993088c4cca31a89efec2ee5 \
 	# Widgets
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/Widgets $MW_HOME/extensions/Widgets \
 	&& cd $MW_HOME/extensions/Widgets \
-	&& git checkout -q 32993157584b182390962d93341561cd3aecdb58 \
+	&& git checkout -q 583c4f5e1829d2f78fb4a6e1f22bc9de4cf4d3a8 \
 	# WikiForum
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/WikiForum $MW_HOME/extensions/WikiForum \
 	&& cd $MW_HOME/extensions/WikiForum \
-	&& git checkout -q 9cffc82dfd761fbb7a91aa778fb6633215c47501 \
+	&& git checkout -q a2685b60af86890f199a5f3b6581918369e6a571 \
 	# WikiSEO
-	&& git clone --single-branch -b master https://gerrit.wikimedia.org/r/mediawiki/extensions/WikiSEO $MW_HOME/extensions/WikiSEO \
+	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/WikiSEO $MW_HOME/extensions/WikiSEO \
 	&& cd $MW_HOME/extensions/WikiSEO \
-	&& git checkout -q 2c0a40267e9e1abd087cf3fd378cc508b8562f9f \
-	# Wiretap
-	&& git clone https://github.com/enterprisemediawiki/Wiretap.git $MW_HOME/extensions/Wiretap \
-	&& cd $MW_HOME/extensions/Wiretap \
-	&& git checkout -q a97b708c3093ea66e7cf625859b1b38178526bab \
-	# WSOAuth (v. 5.0)
-	&& git clone --single-branch -b master https://gerrit.wikimedia.org/r/mediawiki/extensions/WSOAuth $MW_HOME/extensions/WSOAuth \
+	&& git checkout -q 610cffa3345333b53d4dda7b55b2012fbfcee9de \
+	# WSOAuth
+	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/WSOAuth $MW_HOME/extensions/WSOAuth \
 	&& cd $MW_HOME/extensions/WSOAuth \
-	&& git checkout -q 4a08a825b0a667f0a6834f58844af5fd250ceae8 \
+	&& git checkout -q 3c54c4899dd63989bc3214273bf1c5807c7ac5db
+
+# WikiTeq extensions \
```

Comment:
if we want to split the command into two groups, I suggest doing that separately - otherwise its really hard to see which extensions are being removed/added, and which are just being switched to a group

Reply:
It is hard to pull the changes from Canasta if we mix Canasta/Wikiteq extensions. It was my mistake.
Now I want to fix it and separate ones to two groups. I agree, It is hard to see what was changed, but only for this commit. Please, create a task for this if you insist on doing it separately.